taichi/backends/opengl/codegen_opengl.cpp

```diff
@@ -657,7 +657,7 @@ class KernelGen : public IRVisitor {
   }
 
   void visit(IfStmt *if_stmt) override {
-    emit("if ({} != 0) {{", if_stmt->cond->short_name());
+    emit("if (({} & 1) != 0) {{", if_stmt->cond->short_name());
```

Comment:
Currently LLVM backends use -1 for true, while OpenGL backend uses 1 for true, so in afraid of misusing bit_not as logic_not, I have to do this for now.